src/Raven.Server/Documents/Handlers/IndexHandler.cs

```diff
@@ -666,7 +666,7 @@ public async Task SetPriority()
             }
         }
 
-        [RavenAction("/databases/*/indexes/errors", "DELETE", AuthorizationStatus.ValidUser, IsDebugInformationEndpoint = true)]
```

Comment:
I don't like this change.

Reply:
This was clearly a bug. It is a 'DELETE' endpoint not a 'GET' one. Copy/paste issue.